### DISCUSSION TITLE
Adding packages and plugins

### DISCUSSION THREAD

**Discussion by mihailnicamn** [codescandy/Dash-UI#19] [Q&A]
Hello, 

I tried to add DataTables to Dash-UI and it works when I run `gulp` but with `gulp build` it just do not get imported, I can find the packages in `/dist/assets/libs` but still no link into the header. 

Searched if its anything to do with gulp, found no thing (I'm sure that's an easy fix, I didn't found it at the moment)

**Comment by anitaparmar26** [comment]
I think it's a path issue like **Ex: "@@webRoot/"**. If still does not work then share a screenshot where you linked on the page.

**CSS**: `<link href="@@webRoot/node_modules/datatables/media/css/jquery.dataTables.min.css" rel="stylesheet">`
**JS**:   `<script src="@@webRoot/node_modules/datatables/media/js/jquery.dataTables.min.js"></script>`

Thank You

**Comment by mihailnicamn** [comment]
I fixed it by using cdn (as a temp solution) has same issue until I moved the imports below `<!-- endbuild -->` tag.
Has it anything to do with those tags `<!-- Libs JS -->` `<!-- Libs CSS -->` ? (I'm a noob in gulp some reference docs will be helpful)
